src/schema/common.cljs

```diff
@@ -28,10 +28,15 @@
     [:on-success [:or fn? [:cat keyword? [:* :any]]]]
     [:on-error [:or fn? [:cat keyword? [:* :any]]]]]])
 
+(def ^:private ?error
```

Comment:
I'm not sure this schema `?error` captures our future intentions as we mature our error handling practices. My point is that to me, our errors should not necessarily be instances of `js/Error`, but actually instances of `ExceptionInfo`. It would be more clojurey, and we would be able to eventually have a standardized way to consume and generate errors that are not limited by the JS specification.

I'd suggest renaming the schema to `?js-error` instead and leave `?error` for something that's more idiomatic to Clojure.

Reply:
Is this something we're planning to do i.e. moving to clojure exceptions? Were there any discussions in the past I can look into?

Reply:
the schema is now ([cfd9e44](https://github.com/status-im/status-mobile/pull/18756/commits/cfd9e4459bbb7b843c41f3f3ef9026c87510a8db)) named `::exception` and checks against `ExceptionInfo`, since that's what the schema that uses it expects.